src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -42,7 +48,13 @@
     private MenuItem helpMenuItem;
 
     @FXML
-    private StackPane personListPanelPlaceholder;
+    private StackPane travelPlannerPanelPlaceholder;
+
+    @FXML
+    private StackPane travelPlanPanelPlaceholder;
+
+    @FXML
+    private StackPane travelObjectListPanelPlaceholder;
```

Comment:
Same as above. Maybe we put ScrollPane here

Reply:
Same as above too!